lib/Target/CBackend/CBackend.cpp

```diff
@@ -1529,7 +1529,7 @@ static void generateCompilerSpecificCode(raw_ostream& Out,
   Out << "#define __noreturn __declspec(noreturn)\n";
   Out << "#else\n";
   Out << "#define __noreturn __attribute__((noreturn))\n";
-  Out << "#define __forceinline __attribute__((always_inline))\n";
+  Out << "#define __forceinline __attribute__((always_inline)) inline\n";
```

Comment:
Can this be `__inline` and still avoid the gcc warning?

Reply:
why would `__inline` be preferred?

Reply:
To possibly avoid compiler warnings (https://gcc.gnu.org/onlinedocs/gcc/Alternate-Keywords.html#Alternate-Keywords), but doesn't matter